### DISCUSSION TITLE
question about a feature

### DISCUSSION THREAD

**Discussion by forenade** [kimai/kimai#3689] [General]
First of all i want to thank you for a great time tracker and all the work you put in to it.

Then to my question :-)

When my employee makes a new Timesheet and fill in everything that they have done and time. 
Is it possible to make a feature in the bottom at that box to attach photo at the work they have done?
I don't now if this is possible at all but would be a great feature for me and i believe many other workgroups using it to.
I would happily contribute for such feature.

**Comment by kevinpapst** [comment]
This is not possible yet. 

> I would happily contribute for such feature.

What do you understand by that?

> i believe many other workgroups using it to.

I haven't heard that often honestly. Some people asked about such a feature for expenses, but very rarely for timesheets (and only by companies working at construction sites).

**Reply by forenade** [reply]
Ok that's a shame. 

Yes iam in the the construction industry, running s small family business. 

Our employees use the time tracker to report the work they have done and It would be great if you could add pictures.
